src/main/scala/com/databricks/spark/sql/perf/Tables.scala

```diff
@@ -222,7 +222,7 @@ abstract class Tables(sqlContext: SQLContext, scaleFactor: String,
           log.info(s"Data has $numRows rows clustered $clusterByPartitionColumns for $maxRecordPerFile")
 
           if (maxRecordPerFile > 0 && numRows > maxRecordPerFile) {
-            val numFiles = ((numRows)/maxRecordPerFile).ceil.toInt
+            val numFiles = ((numRows * 1.0)/maxRecordPerFile).ceil.toInt
```

Comment:
is it better to call `numRows.toDouble`?

Reply:
Done.